packages/compass-schema/src/modules/schema-analysis.ts

```diff
@@ -85,6 +85,7 @@ export const analyzeSchema = async (
       ns,
       error: err.message,
       aborted: abortSignal.aborted,
+      ...(abortSignal.aborted ? { abortReason: abortSignal.reason } : {}),
```

Comment:
Quite often, in my experience, a `signal.reason` is set to an Error to trace to the cause of the interruption, do we need to pull the instance apart to actually see the message here? (same as `err`?)

Reply:
Updated!